src/restLink.ts

```diff
@@ -84,28 +85,40 @@ export const validateRequestMethodForOperationType = (
   }
 };
 
+let exportVariables = {};
```

Comment:
Must this be a mutated global? Or is there a better way to do this with per request context?

Reply:
There must be something better, I hate globals as anyone I guess